src/formatDate.js

```diff
@@ -8,7 +8,32 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = fromFormat[fromFormat.length - 1];
```

Comment:
The assumption that the last element of `fromFormat` is the separator might not always be correct. Ensure that the format array explicitly includes the separator if this is the intended design.